jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/CardAssert.java

```diff
@@ -31,4 +31,5 @@ public CardAssert assertNumberOfButtonsOnCard(int number) {
         jdiAssert(element().getButtons().size() == number, Matchers.is(true));
         return this;
     }
+
```

Comment:
revert this change, it is another project

Reply:
Please, specify, what exactly should I revert? The whole Card functionality should be returned to DataIterator?

Reply:
This is another project jdi-light-angular, we are working ONLY in vuetify projects and do not use that one